python/pyspark/sql/readwriter.py

```diff
@@ -300,9 +304,12 @@ def table(self, tableName):
         return self._df(self._jreader.table(tableName))
 
     @since(1.4)
-    def parquet(self, *paths):
+    def parquet(self, *paths, **options):
```

Comment:
We need `**options` instead of `recursiveFileLookup=None` to support Python 2.

Reply:
seems fine.